redis/src/cluster.rs

```diff
@@ -586,18 +586,15 @@ where
             }
             Some(ResponsePolicy::Special) | None => {
                 // This is our assumption - if there's no coherent way to aggregate the responses, we just map each response to the sender, and pass it to the user.
-                // TODO - once RESP3 is merged, return a map value here.
                 // TODO - once Value::Error is merged, we can use join_all and report separate errors and also pass successes.
                 let results = results
                     .enumerate()
                     .map(|(index, result)| {
                         let addr = addresses[index];
-                        result.map(|val| {
-                            Value::Array(vec![Value::BulkString(addr.as_bytes().to_vec()), val])
-                        })
+                        result.map(|val| (Value::BulkString(addr.as_bytes().to_vec()), val))
```

Comment:
Why not using SimpleString for the address?

Reply:
I didn't want to change from the previous implementation. You prefer that?